src/pages/Legal/PromoTermsOfUse.vue

```diff
@@ -1,132 +1,90 @@
 <template>
```

Comment:
Removed wrapper elements from this template, all that is left is the actual content to inject into `<router-view/>`